src/lib/create/local.js

```diff
@@ -3,3 +3,7 @@ import { createLocalOrUTC } from './from-anything';
 export function createLocal (input, format, locale, strict) {
     return createLocalOrUTC(input, format, locale, strict, false);
 }
+
+export function createUTC (input, isUTC, format, locale, strict) {
```

Comment:
We already have this function here: https://github.com/moment/moment/blob/develop/src/lib/create/utc.js#L3

Reply:
In the existing createLocal function we are hardcoded the param isUTC, so in compare.js isBefore(), isAfter() and isSame() methods need to pass isUTC while comparing UTC date.
https://github.com/moment/moment/blob/develop/src/lib/moment/compare.js